### DISCUSSION TITLE
SCRIPTABLE OBJECT

### DISCUSSION THREAD

**Discussion by cho66x** [naverz/zepeto-studio-global#46] [General]
how to create exit point inside event trigger object?

**Comment by jhslee** [comment]
Hello @anggacho 👋 

I assume you're talking about the interaction trigger i've sent you. 
Please check the video example here: https://youtu.be/JVFzUX_VJoM?t=193 (3:13) for creating the exit point object. You can position the exit point wherever you'd like, and the character will move to that point when it detects input.

**Comment by cho66x** [comment]
I'm using scale on the sphere collider, is that the source of the problem? @jhslee

**Comment by ckdlskgkdrhd** [comment]
@anggacho are you trying to place the exit point inside a collidered object.
